### PR TITLE
Update inplaceSelect.js

### PR DESCRIPTION
**Pull request by mimmazzo**
RF-11402 - Metamer: rich:inplaceSelect: scroller disappear when move by mouse

### CODE REVIEW

**Comment by mimmazzo** [comment]
Leave a comment

**Comment by lfryc** [comment]
Hey Domenico, could you please improve formatting of source code

to follow original formatting?

I can see the line changes really does not show that formatting would be consistent:
https://github.com/richfaces4/components/pull/82/files#L1L170

**Comment by mimmazzo** [comment]
Ok, i replace tabs with 4 spaces.

**Comment by lfryc** [comment]
Hey Domenico.

I can confim the patch fixes the issue,

but I have few concerns:
- why you have included `inplaceInput` changes in a patch for `inplaceSelect`?
  - could you please open a new issue or find a relevant one for these updates?
- I will comment inline for other concerns..

File: input/ui/src/main/resources/META-INF/resources/org.richfaces/inplaceSelect.js [hunk 1]
```diff
@@ -80,19 +82,9 @@
             },
 
             onsave: function() {
-                var item = this.list.currentSelectItem();
```
Comment (lfryc) [inline input/ui/src/main/resources/META-INF/resources/org.richfaces/inplaceSelect.js:83]: What is an intention of this change? I can't see it relevant to the RF-11030, or is it?

File: input/ui/src/main/resources/META-INF/resources/org.richfaces/inplaceSelect.js [hunk 2]
```diff
@@ -114,15 +105,11 @@
             processItem: function(item) {
                 var label = $(item).data('clientSelectItem').label;
                 this.__setValue(label);
-
-                this.__setInputFocus();
-                this.__hidePopup();
-
-                if (this.saveOnSelect) {
-                    this.save();
+                this.invokeEvent.call(this, "selectitem", document.getElementById(this.id));
```
Comment (lfryc) [inline input/ui/src/main/resources/META-INF/resources/org.richfaces/inplaceSelect.js:108]: why you have changed order of `selectitem` call and saving/hiding?

File: input/ui/src/main/resources/META-INF/resources/org.richfaces/inplaceSelect.js [hunk 3]
```diff
@@ -167,16 +154,12 @@
...
-                    if (this.isEditState()) {
-                        this.timeoutId = window.setTimeout($.proxy(function() {
-                            this.onblur(e);
-                        }, this), 200);
-                    }
-                } else {
-                    this.__setInputFocus();
-                    this.isMouseDown = false;
-                }
+                if (!this.isMouseDown && this.isEditState()) {
```
Comment (lfryc) [inline input/ui/src/main/resources/META-INF/resources/org.richfaces/inplaceSelect.js:157]: Could you describe what was this change necessary?

**Comment by lfryc** [comment]
@mimmazzo could you please comment on my concerns above?

I don't feel confident to merge this pull request since it isn't clear what issue it addresses and how.

**Comment by lfryc** [comment]
Hey @mimmazzo , can I close this pull request? There is no activity for 6 monts now.

**Comment by mimmazzo** [comment]
Sorry, i forget it. Closing.
